easybuild/easyconfigs/g/GCC/GCC-5.2.0-binutils-2.25.eb

```diff
@@ -0,0 +1,55 @@
+name = 'GCC'
+version = '5.2.0'
+
+binutilsver = '2.25'
+versionsuffix = '-binutils-%s' % binutilsver
+
+homepage = 'http://gcc.gnu.org/'
+description = """The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
+ as well as libraries for these languages (libstdc++, libgcj,...)."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+mpfr_version = '3.1.3'
+
+source_urls = [
+    'http://ftpmirror.gnu.org/%(namelower)s/%(namelower)s-%(version)s',  # GCC auto-resolving HTTP mirror
+    'http://ftpmirror.gnu.org/gmp',  # idem for GMP
+    'http://ftpmirror.gnu.org/mpfr',  # idem for MPFR
+    'http://www.multiprecision.org/mpc/download',  # MPC official
+    'ftp://gcc.gnu.org/pub/gcc/infrastructure/',  # GCC dependencies
+    'http://gcc.cybermirror.org/infrastructure/',  # HTTP mirror for GCC dependencies
+]
+
+sources = [
+    SOURCELOWER_TAR_BZ2,
+    'gmp-6.0.0a.tar.bz2',
+    'mpfr-%s.tar.gz' % mpfr_version,
+    'mpc-1.0.3.tar.gz',
+    'isl-0.14.tar.bz2',
+]
+
+patches = [('mpfr-%s-allpatches-20150717.patch' % mpfr_version, '../mpfr-%s' % mpfr_version)]
+
+builddependencies = [
+    ('M4', '1.4.17'),
+    ('binutils', binutilsver),
+]
+
+checksums = [
+    'a51bcfeb3da7dd4c623e27207ed43467',     # gcc-5.2.0.tar.bz2
+    'b7ff2d88cae7f8085bd5006096eed470',     # gmp-6.0.0a.tar.bz2
+    '7b650781f0a7c4a62e9bc8bdaaa0018b',     # mpfr-3.1.l.tar.gz
+    'd6a1d5f8ddea3abd2cc3e98f58352d26',     # mpc-1.0.3.tar.gz
+    'acd347243fca5609e3df37dba47fd0bb',     # isl-0.14.tar.bz2
+    'e502185ebb22b41c528f183bb22a7569',     # mpfr-3.1.3-allpatches-20150717.patch
+]
+
+languages = ['c', 'c++', 'fortran']
+
+withisl = False
```

Comment:
@wpoely86: how come this issue doesn't pop up with `GCC-5.1.0-binutils-2.25.eb`?